partials/verify_login.html

```diff
@@ -5,7 +5,12 @@
 <tr style="page-break-before: always">
   <td valign="top">
     <h1 style="font-family: sans-serif; font-weight: normal; margin: 0 0 24px 0; text-align: center;"><% block verify_title %>{{t "New sign-in to Firefox"}}<% endblock %></h1>
-    <p class="primary" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0 0 24px 0; text-align: center;"><% block verify_content %>{{t "For added security, please confirm this sign-in to begin syncing with %(device)s." }}<% endblock %></p>
+    <p class="primary" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0 0 24px 0; text-align: center;"><% block verify_content %>{{t "For added security, please confirm this sign-in to begin syncing with this device:"}}<br/><br/>
+    {{{ device }}}
+    <span style="display:block;margin:2px 0;">{{t "IP address: %(ip)s" }}</span>
```

Comment:
@TDA - I know this is late in the process - to prepare for phased rollout, can you wrap both the IP address and timestamp lines in `{{if #`'s similar to how `location` is done, and repeat everywhere this info has been added? Can you do that for both the html and text variants of verify_login and new_device_login?